src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
Akira messaged me to elaborate some edge cases we've seen where it's better to use a modifier class on the breadcrumb container. So we're going to do it Akira's way, but let's change the className to `bx--breadcrumb--no-trailing-slash`.

So remove this: 

```css
.bx--breadcrumb-item--last::after{
    content: '';
}
```

And replace with: 

```css
.bx--breadcrumb--no-trailing-slash .bx--breadcrumb-item:last-child::after {
  content: '';
}
```